docs/gistpad.md

```diff
@@ -0,0 +1,51 @@
+# GistPad
+
+[GistPad](https://aka.ms/gistpad) is a Visual Studio Code extension that allows you to edit your GitHub gists and repos, without needing to clone anything locally. It provides support for editing Foam workspaces, complete with `[[link]]` [completion/navigation](https://github.com/vsls-contrib/gistpad#links), [daily pages](https://github.com/vsls-contrib/gistpad#daily-pages), [pasting images](https://github.com/vsls-contrib/gistpad#pasting-images-1) and [backlinks](https://github.com/vsls-contrib/gistpad#backlinks). If you'd like to persist your notes in a GitHub repository, and automatically sync changes without needing to manually commit/push/pull, then GistPad might be an option worth exploring.
+
+<img width="700px" src="https://user-images.githubusercontent.com/116461/87234714-96ba9400-c388-11ea-92c3-544d9a3bb633.png" />
+
+## Getting started
+
+To start using GistPad for your Foam-based knowledge base, simply perform the following steps:
+
+1. Download the [GistPad extension](https://aka.ms/gistpad) and then re-start Visual Studio Code
+
+1. Run the `GistPad: Sign In` command, and provide a GitHub token that includes the `repo` scope (and optionally `gist` and `delete_repo` scope, if you'd like to use GistPad for managing your GitHub content more holistically)
```

Comment:
```suggestion
1. Run the `GistPad: Sign In` command, and provide a [GitHub token](https://github.com/settings/tokens/new) that includes the `repo` scope (and optionally `gist` and `delete_repo` scope, if you'd like to use GistPad for managing your GitHub content more holistically)
```

Might be a good idea to link to the page to generate the token

Reply:
I can say by experience that is better to link to the page. Looking for the token tripped me up for a moment and I had to check the GH docs.